pkg/scanners/scannerv4/scannerv4.go

```diff
@@ -148,6 +153,11 @@ func (s *scannerv4) GetVulnDefinitionsInfo() (*v1.VulnDefinitionsInfo, error) {
 		return nil, fmt.Errorf("pulling metadata from matcher: %w", err)
 	}
 
+	lastTms := metadata.GetLastVulnerabilityUpdate()
+	if lastTms == nil || lastTms.Equal(protoTypes.Timestamp{}) || lastTms.Equal(zeroProtoTimestampFromTime) {
+		return nil, errors.New("no timestamp available")
+	}
+
 	return &v1.VulnDefinitionsInfo{
 		LastUpdatedTimestamp: metadata.GetLastVulnerabilityUpdate(),
```

Comment:
use the variable here, too

Reply:
https://github.com/stackrox/stackrox/pull/9462/commits/5e574092420596352a600bc91f23acf95a75624a